### PR TITLE
Clean tmp files on startup + sent chunks files saving

### PR DESCRIPTION
**Pull request by ddavlet**

### CODE REVIEW

File: src/server/Server.cpp [hunk 1]
```diff
@@ -135,12 +157,40 @@ void Server::setSocketNonblock() {
...
+	new_file_name += "uploads/";
+	new_file_name += oss.str();
+	new_file_name += "-";
+	oss.clear();
+	oss << rand() % 1000;
+	new_file_name += oss.str();
+	new_file_name += ".file";
+
+	rename(file_name.c_str(), new_file_name.c_str());
+	return new_file_name;
```
Comment (ddavlet) [inline src/server/Server.cpp:191]: rename may be not allowed

**Review by lodeme — APPROVED** [review]
lgtm

File: src/server/Server.cpp (hunk 1 above)
Comment (lodeme) [reply src/server/Server.cpp:191]: is this from a particular specification?

Comment (ddavlet) [reply src/server/Server.cpp:191]: subject allows us only certain functions from C, rename is not part of it
as well as remove

**Review by olympiaobr — APPROVED** [review]

**Pull request merged**